test/models/database/post_repository_test.dart

```diff
@@ -0,0 +1,335 @@
+import "package:cloud_firestore/cloud_firestore.dart";
+import "package:fake_cloud_firestore/fake_cloud_firestore.dart";
+import "package:flutter_test/flutter_test.dart";
+import "package:geoflutterfire2/geoflutterfire2.dart";
+import "package:mockito/mockito.dart";
+import "package:proxima/models/database/post_repository.dart";
+import "package:proxima/services/geolocation_service.dart";
+
+import "mock_post_data.dart";
+
+class MockGeoLocationService extends Mock implements GeoLocationService {
```

Comment:
I find a bit weird that is defined here and not in test/services/mock_geo_location_service? However, it is indeed only used in here; so if you don't think any other test will ever need it, I guess it may be fine leaving it here.

Reply:
Yes you are right, we might need it elsewhere, I'll follow your suggestion.
Fix in 06c590e82f11d0092f47bb084db5728c4e3139ae.